### DISCUSSION TITLE
TParams NormalFunction SuspendNormalFunction等需要设置为Public

### DISCUSSION THREAD

**Discussion by danatechgithub4** [junerver/ComposeHooks#13] [Ideas]
![image](https://github.com/junerver/ComposeHooks/assets/127659297/010ee31c-cf0a-46c3-82b6-262c7fe7b7b3)

**Comment by danatechgithub4** [comment]
![image](https://github.com/junerver/ComposeHooks/assets/127659297/207982e1-e629-4527-ae53-43c4a5dfcffb)

useEmptyPlugin是否也可以放开给我们使用？

**Reply by junerver** [reply]
ok，稍后发布15的快照版本，如果有好的idea，欢迎pr

**Comment by junerver** [comment]
因为kt的强类型缺乏足够的灵活性，所以编写自定义插件比较繁琐，而且自定义插件扩展的需求场景也不多。

所以这几个类型别名被我限定在包内使用，我会在快照版本放开这一限制，方便测试，正式版本仍然维持原样，在正式的业务环节你可以复制源码中的 别名声明 到你的文件中。

useEmptyPlugin 将会 public

**Comment by junerver** [comment]
[1.0.15-SNAPSHOT/](https://s01.oss.sonatype.org/content/repositories/snapshots/xyz/junerver/compose/hooks/1.0.15-SNAPSHOT/) 已上传
